server/controllers/monitoringConditions/monitoringConditionsController.ts

```diff
@@ -13,8 +13,30 @@ import nextPage, { getSelectedMonitoringTypes } from './nextPage'
 
 const monitoringConditionsFormDataModel = z.object({
   action: z.string().default('continue'),
-  acquisitiveCrime: z.coerce.boolean(),
-  dapol: z.coerce.boolean(),
+  acquisitiveCrime: z.union([z.string(), z.undefined()]).transform(value => {
+    if (value === undefined) {
+      return null
+    }
+    if (value === 'true') {
```

Comment:
This can be simplified to `return value === 'true'`

Reply:
o the whole thing could just be `value === undefined ? null : value === 'true'`

Reply:
Also, I'm not sure why we'd want to allow a value of anything other than true or false

Reply:
This is an optional field; the user may not want to provide a value for it. As such, the possible inputs are 'true', 'false' and 'undefined'.